Course.java

```diff
@@ -0,0 +1,47 @@
+public class Course {
+  private String courseName;
+  public int classSize =0;
+  private ArrayList<Person> StudentList;
+  private ArrayList<Person> mentorList;
+  public Course(String _courseName){
+    courseName = _courseName;
+
+    StudentList = new ArrayList<Person>();
+    mentorList = new ArrayList<Person>();
+  }
+
+  public void addStudent(Person s){
+    StudentList.add(s);
+    classSize++;
```

Comment:
Consider a better way of getting the length. What if I call `removeStudent` which isn't implemented.